content/concepts/basics/02-content-tree.md

```diff
@@ -4,38 +4,38 @@ metaTitle: "sensenet - Content Tree"
 metaDescription: "sensenet Content Tree"
 ---
 
-sensenet content repository is basically a tree structure of the various stored content (e.g. folders, documents, user, tasks, etc.). A specific content is identified by a unique number (id) and also by its path. The root of the tree is at `/Root` path, all other content is placed somewhere under this root content - for example the users are placed at `/Root/IMS`.
+A sensenet content repository is basically a tree structure of the various stored content (e.g. folders, documents, user, tasks, etc.). A specific content is identified by a unique number (ID) as well as its path. The root of the tree is at `/Root` path, and all other content is placed somewhere under this root content - for example, users are placed at `/Root/IMS`.
 
 # Structured storage of content
 
-Tree structure lets you organize, group your content easily by creating separate subtrees. It's not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of several things like content types' underlying business logic, metadata field configurations and settings.
+The tree structure lets you organize and group your content easily by creating separate subtrees. It is not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of data like content types, underlying business logic, metadata field configurations, and settings.
```

Comment:
```suggestion
The tree structure lets you organize and group your content easily by creating separate subtrees. It is not only important because of the possibility to configure content accessibility for different types of users on different parts of the tree, but also because of the inheritance of data like underlying business logic of content types, metadata field configurations and settings.
```